skeleton/src/main/java/com/dropbox/kaiken/skeleton/scoping/SingleIn.kt

```diff
@@ -120,6 +121,7 @@ interface AuthOptionalScreenComponent : Injector {
     parentScope = UserScope::class
 )
 @SingleIn(AuthRequiredScreenScope::class)
+@MergeComponent(AuthOptionalScreenScope::class)
```

Comment:
AuthRequiredScreenScope

we want this component to have all the objects that do @ContributesBinding(AuthRequiredScreenScope)

This way we can separate what goes into the AuthOptionalScreenComponent from the AuthRequiredScreenComponent

Reply:
actually im not sure this line is needed at all. ContriutsSubcomponent is sort of like a merge component

Reply:
Oops, this was me trying to fix something and forgetting to take it out 😅